web/auth/tokens/jwt.go

```diff
@@ -0,0 +1,156 @@
+package tokens
```

Comment:
We should avoid packages with plural names. So maybe just `token` works? Or maybe it could be `jwt` (I prefer `token`). Another options is to move the content of the `tokens` package into the `auth` package.

Reply:
Moved into the `auth` package.